src/components/OrganizationsTable.tsx

```diff
@@ -3,16 +3,30 @@ import { Table, Spin, Alert } from "antd";
 import { SortOrder } from "antd/lib/table/interface";
 import { Link } from "react-router-dom";
 import { useQuery } from "react-apollo";
-import { fracToPercent } from "utils/displayUtils";
 import {
   IOrganization,
   IAllOrganizations,
   GET_ORGANIZATIONS,
 } from "graphql/queries";
+import { BarType } from "./PercentBar";
+import { PercentageTableCell } from "./PercentageTableCell";
 
 let defaultSort: SortOrder;
 defaultSort = "descend";
 
+const percentColumnRender = (barType: BarType) => {
```

Comment:
It's tricky to figure out what to suggest here, because this depends on a component, but I think a good call is to move it out into a `utils/table.tsx` file. That'll allow you to avoid having two copies of this function.

Reply:
Before going that way (I feel like component-based stuff in `utils` feels off), I'll show (in commit ed48130 here) the other way I thought about doing this. I'm not entirely happy with it, because it means that I'm exporting a function that renders a component alongside the component itself, which doesn't feel right and is at best non-standard. That said, it does fix the "defined in two places" issue. This one was a bit tough, so I'm happy to discuss and iterate on it if you don't like this approach either.